### PR TITLE
(internal) Add token role

### PR DESCRIPTION
**Pull request by mdkulkarni15**

### CODE REVIEW

File: modules/services/service-principal/main.tf [hunk 1]
```diff
@@ -23,7 +23,7 @@ resource "google_project_iam_member" "browser" {
 # role permissions for CSPM (GCP Predefined Roles for Sysdig Cloud Secure Posture Management)
 #---------------------------------------------------------------------------------------------
 resource "google_project_iam_member" "cloudasset_viewer" {
```
Comment (ravinadhruve10) [inline modules/services/service-principal/main.tf:25]: nit: maybe lets change the resource name "`cloudasset_viewer`"?

File: modules/services/service-principal/organizational.tf [hunk 2]
```diff
@@ -26,7 +26,7 @@ resource "google_organization_iam_member" "browser" {
 # role permissions for CSPM (GCP Predefined Roles for Sysdig Cloud Secure Posture Management)
 #---------------------------------------------------------------------------------------------
 resource "google_organization_iam_member" "cloudasset_viewer" {
```
Comment (ravinadhruve10) [inline modules/services/service-principal/organizational.tf:28]: nit: same here

**Review by ravinadhruve10 — APPROVED** [review]
LGTM. Left a non-blocking comment.

File: modules/services/service-principal/main.tf (hunk 1 above)
Comment (mdkulkarni15) [reply modules/services/service-principal/main.tf:25]: naming is hard :) what do we want to call it? cspm?

Comment (ravinadhruve10) [reply modules/services/service-principal/main.tf]: hehe true. :) what you added is good.

**Pull request merged**